PlaygroundTester/Sources/PlaygroundTester/Views/TestMethodsView.swift

```diff
@@ -7,8 +7,11 @@ struct TestMethodsView: View {
   let methods: [ResultViewModel.Suite.Method]
   let title: String
 
+  @Binding private(set) var failuresOnly: Bool
+
   var body: some View {
-    List(methods) { method in
+    let results = failuresOnly ? methods.filter { $0.result.isSuccess == false } : methods
```

Comment:
Hmm, this is a little weird 😅 
This block of code is a regular "block", right? 
How come Swift allows to omit `return` keyword in multiple statement block? 🤔 
Or did they change them to be result builders? (I'm not really super up-to-date with SwiftUI 😅 )